kedro-telemetry/RELEASE.md

```diff
@@ -1,5 +1,9 @@
 # Upcoming release
 
+# Release 0.6.1
+* Changed Kedro CLI loading method to improve loading times.
+* Changed logging level from error to debug for most logging messages.
+
```

Comment:
This one is also worth mentioning https://github.com/kedro-org/kedro-plugins/commit/552b973a256c0f4a9f96e36feb70f4fc15fb371b